docs/index.mdx

```diff
@@ -39,6 +39,14 @@ Each platform provides different features to make an app more accessible. We've
     description="iOS has built-in accessibility features, accessibility APIs, and developer tools which provide an extraordinary opportunity to deliver a superior mobile experience to every user, including those with disabilities."
     image="/img/light/iOS.svg"
     alt="iOS logo"
+  />
+    <LinkListItem
+    url="/en/docs/swiftui"
+    title="Learn about SwiftUI accessibility"
+    label="SwiftUI accessibility"
+    description="SwiftUI has built-in accessibility features, accessibility APIs, and developer tools which provide an extraordinary opportunity to deliver a superior mobile experience to every user, including those with disabilities."
```

Comment:
Change description to: 

SwiftUI automatically provides accessibility support for common elements.  Developers can use SwiftUI's view modifiers to enhance and customize accessibility.